yb-voyager/src/srcdb/pg_dump.go

```diff
@@ -40,8 +40,23 @@ var pgDumpArgs PgDumpArgs
 var pgDumpArgsFile string
 
 func getPgDumpArgsFromFile(sectionToRead string) string {
-	basePgDumpArgsFilePath := filepath.Join("/", "etc", "yb-voyager", "pg_dump-args.ini")
-	if utils.FileOrFolderExists(basePgDumpArgsFilePath) {
+	var basePgDumpArgsFilePath string
+	homebrewVoyagerDir := fmt.Sprintf("yb-voyager@%s", utils.YB_VOYAGER_VERSION)
+	possiblePaths := []string{
+		filepath.Join("/", "etc", "yb-voyager", "pg_dump-args.ini"),
+		filepath.Join("/", "opt", "homebrew", "Cellar", homebrewVoyagerDir, utils.YB_VOYAGER_VERSION, "etc", "yb-voyager", "pg_dump-args.ini"),
+		filepath.Join("/", "usr", "local", "Cellar", homebrewVoyagerDir, utils.YB_VOYAGER_VERSION, "etc", "yb-voyager", "pg_dump-args.ini"),
+	}
+
+	for _, path := range possiblePaths {
+		if utils.FileOrFolderExists(path) {
+			basePgDumpArgsFilePath = path
+			break
+		}
+	}
+	if basePgDumpArgsFilePath == "" {
```

Comment:
you can remove this check. In case if it is not found it will be generated and saved there(just verfiy once).
Because of this code:
```
//go:embed data/pg_dump-args.ini
var pgDumpArgsFile string
```